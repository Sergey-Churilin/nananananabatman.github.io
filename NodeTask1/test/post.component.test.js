import React from 'react';
import renderer from 'react-test-renderer';

import Post from '../src/post.component';

it('renders correctly', () => {
    const post = {
        author: 'author',
        blogId: 0,
        text: 'text'
    },
    tree = renderer
        .create(<Post onClick={() => {}} post={post} key={post.blogId} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
