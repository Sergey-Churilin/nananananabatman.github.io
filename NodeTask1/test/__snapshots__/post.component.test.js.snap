// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="blog-post"
  onClick={[Function]}
>
  <p
    className="blog-post__text"
  >
    text
  </p>
  <p
    className="blog-post__author"
  >
    author
  </p>
</article>
`;
